src/interfaces/models.ts

```diff
@@ -92,12 +92,12 @@ export interface DraftReqI {
 }
 
 export interface EntityI {
-  blurbs?: string[] | string
+  blurbs?: string | string[] | string[][] // TODO: fix wtf here
```

Comment:
This doesn't look right but it's what TS wanted. Have a look.

Ditto on lines 99-100.